Prometheus/NodeExporter.download.recipe

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>Description</key>
+		<string>Downloads the latest Prometheus Node Exporter from GitHub.
+
+Use the ARCH input variable to specify the desired architecture. Values include:
+- x86_64 (default, Intel)
+- arm64 (Apple Silicon)
+</string>
+	<key>Identifier</key>
+	<string>com.github.znerol.download.NodeExporter</string>
+	<key>Input</key>
+	<dict>
+		<key>NAME</key>
+		<string>NodeExporter</string>
+		<key>ARCH</key>
+		<string>x86_64</string>
+	</dict>
```

Comment:
Wouldn't it be appropriate to default to apple silicon instead of intel nowadays?

Reply:
As long as admins are still deploying Rosetta2 broadly, the cost of accidentally deploying the Intel flavor is basically zero, but the cost of accidentally deploying the Apple Silicon flavor is not.

You're right that at some point in the future Apple Silicon by default may be the community expectation.